docker.in

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+[ -e ${ETCDIR}/nodocker ] || \
+echo "Emulate Docker CLI using podman. Create ${ETCDIR}/nodocker to quiet msg." >&2
```

Comment:
Shouldn't this be `${ETCDIR}/`**containers**/nodocker ?

Reply:
Sorry to keep blocking, but line 3 needs the same fix as line 2 (adding **containers** subdirectory).